basics/import_maps.md

```diff
@@ -95,11 +95,11 @@ scope in the import map that looks something like this:
 ```json
 {
   "imports": {
-    "https://deno.land/std@0.177.0/": "https://deno.land/std@$STD_VERSION/"
+    "https://deno.land/std@0.178.0/": "https://deno.land/std@$STD_VERSION/"
```

Comment:
Here I think the version needs to not be the latest, because then the import map just maps to the same version.

Reply:
This makes sense. Reverting the change.